chemprop/cli/utils/parsing.py

```diff
@@ -162,16 +165,16 @@ def make_datapoints(
                 weight=weights[i],
                 gt_mask=gt_mask[i],
                 lt_mask=lt_mask[i],
-                x_f=X_f[i],
+                x_f=X_f[mol_idx][i], # TODO: if X_f contains feature for both molecule and reaction, this is wrong
```

Comment:
Regarding the comment, it says in the PR that we are not supporting features for both a molecule and a reaction. Is this the case, or are we undecided on this?

Reply:
The problem here is that the sequence of molecule and reaction is reordered. Say in the input file the user provides smiles_mol1, smiles_rxn1, smiles_mol2. They also provide additional mol feature in the input file with the sequence of X_f_mol1, X_f_rxn1, X_f_mol2. From `parse_csv`, it combines the smiles for mols into [smiles_mol1, smiles_mol2] and the smiles for rxns into [smiles_rxn1]. We then lose the sequence corresponding to the additional mol feature. It requires another PR to fix this sequence problem.